docs/index.rst

```diff
@@ -129,6 +129,14 @@ The following subsections document each submodule.
    :show-inheritance:
    :member-order: bysource
 
+:mod:`mir_eval.multipitch`
```

Comment:
Hopefully multipitch is an [intentional reference](https://en.wikipedia.org/wiki/Multi-pitch_climbing)...